entity/pokemon.go

```diff
@@ -0,0 +1,17 @@
+package entity
+
+type Pokemon struct {
+	Id         int    `json:"id"`
```

Comment:
Acronyms should be written in ALL CAPS.
```suggestion
	ID         int    `json:"id"`
```